docs/hiring-guide/community-spotlight/interview-liz-harris.rst

```diff
@@ -0,0 +1,141 @@
+    exchange student in Switzerland for a year, and then I was going to do four years at this small college 
+    in Vermont. But when I finished my exchange student year, I realized that this small college in Vermont 
+    was not going to work for me. It's a good school, but it was way too small and socially limited and 
+    academically limited. So I did my freshman year at Middlebury College, and then I transferred to a 
+    bigger school. I transferred to Harvard and I finished my bachelor's degree.  And after, I went to 
+    graduate school, kind of without really thinking about it. My family is full of professors, so that's 
+    kind of just the family business. I mean, when we have Thanksgiving dinner, there will be like 12 people 
+    at the table with two who don't have a PhD. It's ridiculous. So I went to graduate school in Russian 
+    literature at Stanford, and I thought I wanted to get a PhD in Russian literature. But that was a 
+    terrible experience. It was one of the worst times of my life. I hated it. And so after two years, I 
+    finished with two different masters degrees and stepped out and went and got a teaching job.
+
+
+What then led you to technical writing?
+
+    Well, I loved teaching. I was a teacher at a private school in San Francisco, at a French school. 
+    And loved it, but it was the late nineties. There was a tech boom going on in the Bay Area, and I 
+    wasn't participating in it. There was a lot of money being made, and it was so easy to get a job. 
+    The hiring was ferocious. There were people who I knew from growing up in the area working in tech, 
+    who I knew were no smarter than or no more hardworking than me, and they had these jobs that were better 
+    paid than a teacher's job, but also interesting and appealing. The only reason I had heard of the 
+    profession of a technical writer was from a stepmother who was in the family for a while. She was a 
+    technical writer. She actually was a poet and an English teacher, and then she became a technical 
+    writer in the eighties. So I think it was 1998. I had tried to stay in teaching, but I didn't have a 
+    teaching credential. The reason I was looking for a teaching job is I really like teaching. But my 
+    particular school where I was working was a dysfunctional mess. So after three years in a row of not 
+    finding a different teaching job, I thought forget it. I'm just going to do something else. And so I 
+    took a huge career leap. No, it's not that much of a career leap when you're 27 and don't have a mortgage 
+    and don't have children. There wasn't that much at stake. But my husband was in graduate school, so my 
+    teaching job was paying the rent. It was a bit of a leap in that way. So I put out an application or two 
+    and got a job.
+
+
+Liz's first job as a tech writer involved creating documentation for startup called Prototype that 
```

Comment:
We should probably **bold** the questions, or add some highlighting here, to show the difference between a question you asked and a summary.